eventstore/models.py

```diff
@@ -160,3 +160,27 @@ class PostbirthRegistration(models.Model):
     timestamp = models.DateTimeField(default=timezone.now)
     created_by = models.CharField(max_length=255, blank=True, default="")
     data = JSONField(default=dict, blank=True, null=True)
+
+
+class Messages(models.Model):
```

Comment:
It's considered Django best practice to only name models using the singular, not plural: https://wsvincent.com/django-models-best-practices/

Reply:
Okay, thanks for the resource